tests/func-tests/node_placement_test.go

```diff
@@ -158,3 +244,26 @@ func getNetworkAddonsConfigs(client kubecli.KubevirtClient) *networkaddonsv1.Net
 
 	return &cnaoCR
 }
+
+func setHcoNodeTypeLabel(client kubecli.KubevirtClient, node *v1.Node, value string) error {
+	labels := node.GetLabels()
+	labels[hcoLabel] = value
+	conflict := false
+	for {
+		node, err := client.CoreV1().Nodes().Get(context.TODO(), node.Name, k8smetav1.GetOptions{})
```

Comment:
Because I'm trying to get a specific node and not all the nodes in the cluster. And the reason I have put it in a `for` loop is because of the resourceVersion related error which prevent the `Update()` a couple of lines later because something else on the k8s cluster made some change to the node resource.

Reply:
In that case, please check `Eventually`.